src/utils/line.ts

```diff
@@ -430,3 +433,56 @@ const _createTokenView = (tokenResponse: TokenFragRepsonse, amount?: BigNumber,
     description: '',
   };
 };
+
+export const formatGetBorrowerQuery = (data: AggregatedCreditLine[], borrowerAddress: Address) => {
+  let aggregate: CreditLinePage = {
+    borrower: borrowerAddress,
+    principal: '0', //done
+    deposit: '0', //done
+    highestApy: ['0', '0', '0'], //todo
+    positions: [], //done
+    escrow: {
+      id: '',
+      cratio: '',
+      minCRatio: '',
+      collateralValue: '',
+      //@ts-ignore
```

Comment:
what is this `@ts-ignore` solving?

Reply:
const status = position.status === 'OPEN' ? 'OPENED' : position.status;
 
Type issue that I will tackle today.

Types for line statuses are OPENED, PROPOSED AND LIQUIDATABLE

but OPEN status is being returned from subgraph.

Reply:
So there should be an empty array of EscrowDepositList, type there. But since I am initializing the object with an empty array i get the Never[] does not match EscrowDepositList[] error

Since I am in an obj definition that's typed I can't type the key: value pair